### PR TITLE
added an example for the set operation union all

### PR DESCRIPTION
**Pull request by joemoe**
* three tables
* one union
* some documentation
* a screenshot

### CODE REVIEW

File: set-operations/01/01_union-all.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+SELECT visitor, '' AS spacecraft, location, visit_time FROM rickandmorty_visits
+UNION ALL
+SELECT '' AS visitor, spacecraft, location, visit_time FROM spaceagency_visits
+UNION ALL
+SELECT visitor, starship AS spacecraft, location, visit_time FROM hitchhiker_visits;
+```
+
+## The Beauty in VVP
+
+![01_union_all](screeny.png)
```
Comment (knaufk) [inline set-operations/01/01_union-all.md:77]: I guess a screenshot of executing the UNION query would be nicer.

File: set-operations/01/01_union-all.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+There are three sources of visits. The univers of Rick and Morty, the very real world of NASA and such, 
+and the not so real world of Hitchhikers Guide To The Galaxy.
+
+All three tables are `unbound` and backed by the [`faker` connector](https://github.com/knaufk/flink-faker).
+
+All sources of tracked visits have the `location` and `visit_time` in commmon. Some have `visitors` some have
+`spacecrafts` and one has both.
+
+```sql
+DROP TABLE IF EXISTS rickandmorty_visits;
```
Comment (knaufk) [inline set-operations/01/01_union-all.md:21]: I am more of a fan of self-contained scripts with CREATE TEMPORARY TABLE statements, but we have not aligned on anything yet.

File: README.md [hunk 3]
```diff
@@ -43,6 +43,10 @@ The cookbook is a living document. :seedling:
 5. [Star Schema Denormalization (N-Way Join)](joins/05/05_star_schema.md)
 6. [Lateral Table Join](joins/06/06_lateral_join.md)
 
+### Set Operations
+
+1. [UNION ALL](set-operations/01/01_union-all.md)
```
Comment (knaufk) [inline README.md:48]:
```suggestion
1. [Building the Union of Multiple Streams](set-operations/01/01_union-all.md)
```

What do you think about expanding the title a bit?

**Review by knaufk — COMMENTED** [review]
Thanks @joemoe! Generally looks good. Some suggestions.

File: set-operations/01/01_union-all.md (hunk 2 above)
Comment (joemoe) [reply set-operations/01/01_union-all.md:21]: `TEMPORARY` seems to be undocumented for `CREATE TABLE`.
https://ci.apache.org/projects/flink/flink-docs-master/docs/dev/table/sql/create/#create-table
i guess i'd leave it as it is.

**Comment by joemoe** [comment]
@knaufk committed a fixup based on your review

**Comment by joemoe** [comment]
@knaufk now you got everything you ever wanted. can you have another look?

File: set-operations/01/01_union-all.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+:bulb: This example will show how you can use the set operation `UNION ALL` to combine several streams of data.
+
+See [our documentation](https://ci.apache.org/projects/flink/flink-docs-master/docs/dev/table/sql/queries/#set-operations)
+for a full list of fantastic set operations Apache Flink supports.
+
+
+## The Sources
+
+The examples assumes you are building an application that is tracking visits :raccoon: on foreign planets :chestnut:. 
+There are three sources of visits. The univers of Rick and Morty, the very real world of NASA and such,
```
Comment (sjwiesman) [inline set-operations/01/01_union-all.md:12]:
```suggestion
There are three sources of visits. The universe of Rick and Morty, the very real world of NASA and such, 
```

**Review by knaufk — APPROVED** [review]
LGTM

File: README.md [hunk 5]
```diff
@@ -43,6 +43,10 @@ The cookbook is a living document. :seedling:
 5. [Star Schema Denormalization (N-Way Join)](joins/05/05_star_schema.md)
 6. [Lateral Table Join](joins/06/06_lateral_join.md)
 
+### Set Operations
```
Comment (morsapaes) [inline README.md:46]: I don't think this deserves a new category. How many more recipes will we write on set operations? As we discussed before, this is not supposed to fully emulate the docs. 😉 

What about expanding "Other Built-in Functions" to "Other Built-in Functions & Operators"?

**Pull request merged**